common/browser_process.go

```diff
@@ -179,41 +179,51 @@ func execute(
 	return command{cmd, done, stdout, stderr}, nil
 }
 
-// parseDevToolsURL grabs the websocket address from chrome's output and returns it.
-func parseDevToolsURL(ctx context.Context, cmd command) (wsURL string, _ error) {
-	type result struct {
-		devToolsURL string
-		err         error
-	}
-	c := make(chan result, 1)
+// parseDevToolsURL grabs the WebSocket address from Chrome's output and returns
+// it. If the process ends abruptly, it will return the first error from stderr.
+func parseDevToolsURL(ctx context.Context, cmd command) (string, error) { //nolint: cyclop
```

Comment:
I made the goroutine lambda smaller, in order to react on `cmd.done` in the same `select`, but it caused a slight increase in cyclomatic complexity, and found it simpler to disable the linter here.

I actually think the current version is more readable, but let me know if you prefer refactoring it in some way.